packages/modeling/src/maths/mat4/fromTaitBryanRotation.js

```diff
@@ -13,7 +13,7 @@ const { sin, cos } = require('../utils/trigonometry')
  * @returns {mat4} out
  * @alias module:modeling/maths/mat4.fromTaitBryanRotation
  * @example
- * let matrix = fromTaitBryanRotation(create(), Math.PI / 2, 0, Math.PI)
+ * let matrix = fromTaitBryanRotation(create(), TAU / 4, 0, TAU / 2)
```

Comment:
I see a lot of PI/2 computations. You could declare a constant for that as well (no cool sounding name, unfortunately). It would have even more performance impact since division is much more costly than multiplication.